recipes/jinja2cpp/all/patches/fix-cmake-1.2.1.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index bf603b0..6229f02 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -282,7 +282,7 @@ install(
+         InstallTargets
+     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/static
++    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+     PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/jinja2cpp
+ )
+ 
+diff --git a/src/helpers.h b/src/helpers.h
+index 3af280f..a6a3205 100644
+--- a/src/helpers.h
++++ b/src/helpers.h
+@@ -32,12 +32,21 @@ struct MultiStringLiteral
+ #endif
+     }
+ 
++#ifdef _WIN32
```

Comment:
maybe `_MSC_VER` is more accurate here?

Reply:
Definitely. I was going to ask how to do this, thanks.